client-config-stage.json

```diff
@@ -1,25 +1,20 @@
 {
-    "defaultPool": "BOB2USDC-goerli",
+    "defaultPool": "BOB-sepolia",
 
     "pools": {
         "BOB-sepolia": {
             "chainId": 11155111,
-            "poolAddress": "0x3bd088C19960A8B5d72E4e01847791BD0DD1C9E6",
+            "poolAddress": "0x77f3D9Fb578a0F2B300347fb3Cd302dFd7eedf93",
```

Comment:
I'm not sure about the complete abandoning of this pool. It is the oldest pool and I think it is worth keeping it for backward compatibility testing, stress testing, and so on. What do you think?